src/components/pages/MatchUp/RenderMatchUp.js

```diff
@@ -19,7 +19,15 @@ const RenderMatchUp = props => {
     <>
       <Header displayMenu={true} title="Scribble Stadium" />
       {/**Temporary button to redirect to point share component */}
-      <Button onClick={goToPointShare}>Go To Point Share</Button>
+      <div style={{ textAlign: 'center' }}>
+        <Button
+          style={{ marginTop: '1rem' }}
```

Comment:
similar to above question about in-line styles. I see that the first button had the in-line initially.